distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Aggregate_Column_Helper.enso

```diff
@@ -0,0 +1,62 @@
+from Standard.Base import all
+
+from Standard.Table.Data.Table as Table_Module import No_Such_Column_Error
+from Standard.Table.Data.Column as Column_Module import Column
+import Standard.Table.Data.Aggregate_Column
+import Standard.Table.Internal.Unique_Name_Strategy
+
+from Standard.Table.Error as Error_Module import Missing_Input_Columns, Column_Indexes_Out_Of_Range, No_Output_Columns, Duplicate_Output_Column_Names, Invalid_Output_Column_Names
+
+## PRIVATE
+   Output of validation process for aggregate columns
+type Validated_Aggregate_Columns
+    type Validated_Aggregate_Columns key_columns valid_columns mapped_names problems
```

Comment:
Would be good to add types and shortly describe what each field represents here.

Reply:
For example I'd like to know from the doc that `mapped_names` are matching the entries from `valid_columns`.